### PR TITLE
Use given instead of implicit

### PR DESCRIPTION
**Pull request by nicolasstucki**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -42,8 +42,7 @@
 	url = https://github.com/dotty-staging/scala-xml
 [submodule "community-build/community-projects/shapeless"]
 	path = community-build/community-projects/shapeless
-	url = https://github.com/milessabin/shapeless
-	branch = shapeless-3
+	url = https://github.com/dotty-staging/shapeless
```
Comment (nicolasstucki) [inline .gitmodules]: @milessabin, to make some small changes to shapeless I forked it and used a new branch. I will upstream those changes once this PR is merged.

**Comment by smarter** [comment]
Maybe best to wait until https://github.com/lampepfl/dotty/issues/7151 is resolved before merging this ?

**Comment by nicolasstucki** [comment]
I would say we should not wait. This PR modified only a few definitions using `given`. It misty updates the use sites by adding the missing imports.

**Comment by odersky** [comment]
I vote for holding off. If we change the syntax we will disallow the old given syntax, so we will have a major migration hassle on our hands.

**Comment by nicolasstucki** [comment]
Ok, then I'll reopen this PR when #7151 is in.

**Pull request closed without merging**